vermouth/ffinput.py

```diff
@@ -409,7 +409,7 @@ def _modification_atoms(self, line, lineno=0):
         tokens = collections.deque(_tokenize(line))
         _parse_link_atom(tokens, self.current_modification,
                          defaults={'PTM_atom': False},
-                         treat_prefix=False)
+                         treat_prefix=True)
```

Comment:
Why this change?

Reply:
because otherwise you end up with some nodes having the order attribute and others not having the order attribute.

Reply:
but apparently it messes with some part of the code

Reply:
Ok. I guess that will get fixed at some point in the not-so-near future when we redo the parsers again.

Reply:
@pckroon can you see why this breaks the code? We either have the option to see why the order attribute on modifications messes with the code or we have to copy the entire base parser for interactions again only for modifications. The order attribute is so hard-coded into the interactions parser it is not even funny anymore. This is the last PR that has to be done for publication.